src/video/x11/SDL_x11video.c

```diff
@@ -481,6 +479,7 @@ void X11_VideoQuit(SDL_VideoDevice *_this)
     }
 #endif
 
+    X11_QuitPen(_this);
```

Comment:
Suggest moving this down with the other input quit functions, maybe after QuitTouch, unless there's an order dependency?

Reply:
Fixed in 0ff84e6b8a42d5282ea442ea264409f1698d8f2f.